index.html

```diff
@@ -940,8 +940,18 @@ <h2 class='informative'>Vary</h2>
       <section id="transmission-fixity" class="informative">
         <h2>Transmission Fixity</h2>
         <p>
-          Transmission fixity is verified by application of the <code>Digest</code> header
-          defined in [[RFC3230]] to <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+          Transmission fixity may be verified by including a <code>Digest</code> header (defined in
+          [[RFC3230]]) in <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+        </p>
+      </section>
+      <section id="persistence-fixity" class="informative">
+        <h2>Persistence Fixity</h2>
+        <p>
+          Persistence fixity may be verified by retrieving the current checksum of an <a>LDP-NR</a> and
```

Comment:
@birkland I guess what I'm saying is that the important thing is that the content can be retrieved, and is the same content as some other point in time.  So drawing the connection to the internal state of the repository seems like it's going beyond what the spec should be concerned with, where implementations may reasonably differ.